shared/applicability/cpes.yml

```diff
@@ -0,0 +1,325 @@
+cpes:
+  "cpe:/o:centos:centos:6":
+    id: centos6
+    title: "CentOS 6"
+    check_id: installed_OS_is_centos6
+
+  "cpe:/o:centos:centos:7":
+    id: centos7
+    title: "CentOS 7"
+    check_id: installed_OS_is_centos7
+
+  "cpe:/o:centos:centos:8":
+    id: centos8
+    title: "CentOS 8"
+    check_id: installed_OS_is_centos8
+
+  "cpe:/a:google:chromium-browser":
+    id: chromium
+    title: "Google Chromium Browser"
+    check_id: installed_app_is_chromium
+
+  "cpe:/a:chrony":
+    id: chrony
+    title: "Package chrony is installed"
+    check_id: installed_env_has_chrony_package
+
+  "cpe:/a:container":
+    id: container
+    title: "Container"
+    check_id: installed_env_is_a_container
+
+  "cpe:/o:debianproject:debian:9":
+    id: debian9
+    title: "Debian release 9 (Stretch)"
+    check_id: installed_OS_is_debian9
+
+  "cpe:/o:debian:debian_linux:10":
+    id: debian10
+    title: "Debian Linux 10"
+    check_id: installed_OS_is_debian10
+
+  "cpe:/o:example":
+    id: example
+    title: "Example"
+    check_id: installed_OS_is_part_of_Unix_family
+
+  "cpe:/o:fedoraproject:fedora:32":
+    id: fedora32
+    title: "Fedora 32"
+    check_id: installed_OS_is_fedora
```

Comment:
This should be in the product.yml file.

Reply:
Yeah, It think it would be better placed at the `product.yml`.
But I didn't want to overhaul it all at once.

This first step is just to unify the source of the CPEs.

Reply:
It is better to overhaul and move the CPEs into `product.yml` and move the shared cpes into the `shared/applicability/cpes.yml` now.

Reply:
Otherwise, down with these changes btw as this helps to make the content easier to use for sure.

Reply:
Moved them.